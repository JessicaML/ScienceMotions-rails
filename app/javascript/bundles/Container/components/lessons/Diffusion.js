import React from "react";
import DiffusionScript from "./DiffusionScript";
import Text from "./text/Text";

const Diffusion = props => {
  return (
      <section>
        <div className="diff">
          <DiffusionScript />
        </div>
        <div>
          <Text props={props.description} />
        </div>
      </section>
  );
};
export default props => <Diffusion {...props} />;
