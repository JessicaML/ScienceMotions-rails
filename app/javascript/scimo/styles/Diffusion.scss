.diffusionWrapper {
    height: 300px;

    .buttonWrapper {
        position: absolute;
        top: 75px;
        left: 20px;    
    }
}


@media only screen and (max-width: 767px) {

    .diffusionWrapper {
        height: 350px;
    
        .buttonWrapper {
            position: absolute;
            top: 125px;
            left: 20px;    
        }
    }
}