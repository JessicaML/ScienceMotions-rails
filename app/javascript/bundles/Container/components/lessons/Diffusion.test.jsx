import React from 'react';
import renderer from 'react-test-renderer';
import Diffusion from './Diffusion';

describe('Diffusion', () => {
  window.scrollTo = jest.fn(() => ({})); 
  it('renders correctly when there are no items', () => {
    const rendered = renderer.create(<Diffusion />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
})