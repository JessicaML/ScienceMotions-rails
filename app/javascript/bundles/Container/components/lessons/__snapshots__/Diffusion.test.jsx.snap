// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Diffusion renders correctly when there are no items 1`] = `
<section>
  <div
    className="diff"
  >
    <div
      id="diffusion"
    />
  </div>
  <div>
    <section
      id="text"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": undefined,
          }
        }
      />
    </section>
  </div>
</section>
`;
